position_based_image/html/index.js

```diff
@@ -0,0 +1,144 @@
+const areaForms = [];
+const buttons = [];
+
+$(document).ready(function () {
+    $.get('/local/camscripter/package/settings.cgi?package_name=position_based_image&action=get', function (settings) {
+        $('#scIP').val(settings.sourceCamera.IP);
```

Comment:
Je tu spoustu attributu, ktere se nepouzivaji a ani nejsou v UI. To same v settings.json a v main.js

Reply:
!